dracut/30ignition/coreos-gpt-setup.sh

```diff
@@ -6,8 +6,18 @@ set -euo pipefail
 
 UNINITIALIZED_GUID='00000000-0000-4000-a000-000000000001'
 
-# PTUUID is the disk guid, PKANME is the parent kernel name
-eval $(lsblk --output PTUUID,PKNAME --pairs --paths --nodeps "$1")
+# On RHEL 8 the version of lsblk doesn't have PTUUID. Remove this
+# 'if' and just use the 'else' in the future.
+if [ "$(lsblk --version)" == 'lsblk from util-linux 2.32.1' ]; then
```

Comment:
How about detecting the feature instead with `lsblk --help | grep -q PTUUID`?

Reply:
updated!